test/TitleEscrowCreator.js

```diff
@@ -15,11 +15,12 @@ contract("TitleEscrowCreator", accounts => {
   let TitleEscrowCreatorInstance;
 
   before(async () => {
-    TitleEscrowCreatorInstance = await TitleEscrowCreator.new();
+    // TitleEscrowCreatorInstance = await ERC721.new();
   });
 
   beforeEach(async () => {
     ERC721Instance = await ERC721.new("foo", "bar");
+    TitleEscrowCreatorInstance = ERC721Instance
```

Comment:
IF `ERC721Instance === TitleEscrowCreatorInstance` why not just keep one of it?

Reply:
done. removed TitleEscrowCreatorInstance and replaced anywhere used with ERC721Instance.